Documentation/installation/servicemesh/ingress.rst

```diff
@@ -24,6 +24,32 @@ an existing K8s cluster with Cilium installed.
 
 .. include:: installation.rst
 
+Supported Ingress Annotations
+#############################
+
+.. list-table:: 
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Default Value
+   * - ``io.cilium/tcp-keep-alive``
+     - Enable TCP keep-alive
+     - 1 (enabled)
+   * - ``io.cilium/tcp-keep-alive-idle``
+     - TCP keep-alive idle time (in seconds)
+     - 10s
+   * - ``io.cilium/tcp-keep-alive-probe-interval``
+     - TCP keep-alive probe intervals (in seconds)
+     - 5s
+   * - ``io.cilium/tcp-keep-alive-probe-max-failures``
+     - TCP keep-alive probe max failures
+     - 10
+   * - ``io.cilium/websocket``
+     - Enable websocket
```

Comment:
“websocket” is not recognised by the spell checker, you should add it to the list of exceptions

![image](https://user-images.githubusercontent.com/17001771/185627220-093612bd-6165-494e-9625-9e8345a69418.png)